terraform/sitesusa.app.cloud.gov.tf

```diff
@@ -0,0 +1,384 @@
+  type = "CNAME"
+  ttl = 60
+  records = ["d2zeuk52vop2m1.cloudfront.net."]
+}
+
+resource "aws_route53_record" "citizenscience_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "citizenscience.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d14h3zbfoxnlbd.cloudfront.net."]
+}
+
+resource "aws_route53_record" "communitysolutions_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "communitysolutions.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d2shhl1m4kf8sy.cloudfront.net."]
+}
+
+resource "aws_route53_record" "akamaitest_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "akamaitest.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d25p97zz7oomej.cloudfront.net."]
+}
+
+resource "aws_route53_record" "blm_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "blm.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d1jo0f6cbg1d05.cloudfront.net."]
+}
+
+resource "aws_route53_record" "mip_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "mip.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d3937bwai69yxa.cloudfront.net."]
+}
+
+resource "aws_route53_record" "privacy_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "privacy.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d2moi27garks9e.cloudfront.net."]
+}
+
+resource "aws_route53_record" "cio_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "cio.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d3l9veg4o1omhy.cloudfront.net."]
+}
+
+resource "aws_route53_record" "wellness_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "wellness.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d3muasq47exin3.cloudfront.net."]
+}
+
+resource "aws_route53_record" "trust_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "trust.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d1wqojscappq8s.cloudfront.net."]
+}
+
+resource "aws_route53_record" "taxdesignchallenge_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "taxdesignchallenge.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d1yrz6cljgdind.cloudfront.net."]
+}
+
+resource "aws_route53_record" "nidanrb_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "nidanrb.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d3chp99iuj1c5n.cloudfront.net."]
+}
+
+
+output "cloud_gov_ns" {
```

Comment:
This named variable should be renamed to `sitesusa_app_cloud_gov_ns` instead of `cloud_gov_ns`.